### PR TITLE
feat: add k8s components

### PR DESCRIPTION
**Pull request by josedonizetti**
Signed-off-by: Jose Donizetti <jdbjunior@gmail.com>

## Description

Adds a new table with checks for infra assesment.

```
trivy k8s all --report=summary --namespace=kube-system
34 / 34 [--------------------------------------------------------------------------------------------------------------------] 100.00% 1 p/s

Summary Report for minikube
┌─────────────┬──────────────────────────────────────┬─────────────────────┬────────────────────┬───────────────────┐
│  Namespace  │               Resource               │   Vulnerabilities   │ Misconfigurations  │      Secrets      │
│             │                                      ├───┬────┬───┬────┬───┼───┬───┬───┬────┬───┼───┬───┬───┬───┬───┤
│             │                                      │ C │ H  │ M │ L  │ U │ C │ H │ M │ L  │ U │ C │ H │ M │ L │ U │
├─────────────┼──────────────────────────────────────┼───┼────┼───┼────┼───┼───┼───┼───┼────┼───┼───┼───┼───┼───┼───┤
│ kube-system │ Deployment/coredns                   │ 1 │ 2  │ 1 │ 1  │ 4 │   │   │ 3 │ 5  │   │   │   │   │   │   │
│ kube-system │ Pod/etcd-minikube                    │   │ 16 │ 4 │    │ 4 │   │ 1 │ 3 │ 7  │   │   │   │   │   │   │
│ kube-system │ Pod/kube-scheduler-minikube          │   │ 1  │   │    │   │   │ 1 │ 3 │ 8  │   │   │   │   │   │   │
│ kube-system │ Pod/storage-provisioner              │   │ 8  │ 2 │    │ 2 │   │ 1 │ 5 │ 10 │   │   │   │   │   │   │
│ kube-system │ DaemonSet/kube-proxy                 │   │ 2  │ 2 │ 22 │   │   │ 2 │ 4 │ 10 │   │   │   │   │   │   │
│ kube-system │ Pod/kube-controller-manager-minikube │   │ 1  │ 1 │ 1  │ 2 │   │ 1 │ 3 │ 8  │   │   │   │   │   │   │
│ kube-system │ Service/kube-dns                     │   │    │   │    │   │   │   │ 1 │    │   │   │   │   │   │   │
│ kube-system │ Pod/kube-apiserver-minikube          │   │ 1  │ 1 │ 1  │ 2 │   │ 1 │ 3 │ 9  │   │   │   │   │   │   │
└─────────────┴──────────────────────────────────────┴───┴────┴───┴────┴───┴───┴───┴───┴────┴───┴───┴───┴───┴───┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN


Summary Report for minikube
┌─────────────┬─────────────────────────────────────────────────────┬───────────────────┐
│  Namespace  │                      Resource                       │  RBAC Assessment  │
│             │                                                     ├───┬───┬───┬───┬───┤
│             │                                                     │ C │ H │ M │ L │ U │
├─────────────┼─────────────────────────────────────────────────────┼───┼───┼───┼───┼───┤
│ kube-system │ Role/system::leader-locking-kube-controller-manager │   │   │ 1 │   │   │
│ kube-system │ Role/system:controller:bootstrap-signer             │ 1 │   │   │   │   │
│ kube-system │ Role/system:controller:cloud-provider               │   │   │ 1 │   │   │
│ kube-system │ Role/system:controller:token-cleaner                │ 1 │   │   │   │   │
│ kube-system │ Role/system:persistent-volume-provisioner           │   │ 2 │   │   │   │
│ kube-system │ Role/system::leader-locking-kube-scheduler          │   │   │ 1 │   │   │
└─────────────┴─────────────────────────────────────────────────────┴───┴───┴───┴───┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN


Summary Report for minikube
┌─────────────┬──────────────────────────────────────┬─────────────────────────────┐
│  Namespace  │               Resource               │ Kubernetes Infra Assessment │
│             │                                      ├────┬────┬────┬─────┬────────┤
│             │                                      │ C  │ H  │ M  │ L   │   U    │
├─────────────┼──────────────────────────────────────┼────┼────┼────┼─────┼────────┤
│ kube-system │ Pod/kube-apiserver-minikube          │    │    │ 1  │ 10  │        │
│ kube-system │ Pod/kube-controller-manager-minikube │    │    │    │ 3   │        │
│ kube-system │ Pod/kube-scheduler-minikube          │    │    │    │ 1   │        │
└─────────────┴──────────────────────────────────────┴────┴────┴────┴─────┴────────┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN
```

filtering by workload components:

```
trivy k8s all --report=summary --namespace=kube-system --security-checks=config --components=workload
34 / 34 [--------------------------------------------------------------------------------------------------------------------] 100.00% 4 p/s

Summary Report for minikube
┌─────────────┬──────────────────────────────────────┬────────────────────┐
│  Namespace  │               Resource               │ Misconfigurations  │
│             │                                      ├───┬───┬───┬────┬───┤
│             │                                      │ C │ H │ M │ L  │ U │
├─────────────┼──────────────────────────────────────┼───┼───┼───┼────┼───┤
│ kube-system │ Service/kube-dns                     │   │   │ 1 │    │   │
│ kube-system │ Pod/kube-apiserver-minikube          │   │ 1 │ 3 │ 9  │   │
│ kube-system │ Pod/kube-scheduler-minikube          │   │ 1 │ 3 │ 8  │   │
│ kube-system │ Pod/storage-provisioner              │   │ 1 │ 5 │ 10 │   │
│ kube-system │ Pod/etcd-minikube                    │   │ 1 │ 3 │ 7  │   │
│ kube-system │ Deployment/coredns                   │   │   │ 3 │ 5  │   │
│ kube-system │ DaemonSet/kube-proxy                 │   │ 2 │ 4 │ 10 │   │
│ kube-system │ Pod/kube-controller-manager-minikube │   │ 1 │ 3 │ 8  │   │
└─────────────┴──────────────────────────────────────┴───┴───┴───┴────┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN
```

filtering by infra components:

```
trivy k8s all --report=summary --namespace=kube-system --security-checks=config --components=infra
34 / 34 [--------------------------------------------------------------------------------------------------------------------] 100.00% 4 p/s

Summary Report for minikube
┌─────────────┬──────────────────────────────────────┬─────────────────────────────┐
│  Namespace  │               Resource               │ Kubernetes Infra Assessment │
│             │                                      ├────┬────┬────┬─────┬────────┤
│             │                                      │ C  │ H  │ M  │ L   │   U    │
├─────────────┼──────────────────────────────────────┼────┼────┼────┼─────┼────────┤
│ kube-system │ Pod/kube-scheduler-minikube          │    │    │    │ 1   │        │
│ kube-system │ Pod/kube-apiserver-minikube          │    │    │ 1  │ 10  │        │
│ kube-system │ Pod/kube-controller-manager-minikube │    │    │    │ 3   │        │
└─────────────┴──────────────────────────────────────┴────┴────┴────┴─────┴────────┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN

```

## Related issues
- Close https://github.com/aquasecurity/trivy/issues/2766

Remove this section if you don't have related PRs.

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [x] I've added usage information (if the PR introduces new options)
- [x] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

**Comment by knqyf263** [comment]
@chen-keinan Can you please review it?

**Comment by josedonizetti** [comment]
@knqyf263 @chen-keinan I'm changing it currently, will be ready for review soon.

**Comment by josedonizetti** [comment]
@chen-keinan can you review it? pls

File: pkg/k8s/report/report.go [hunk 1]
```diff
@@ -237,3 +283,60 @@ func (r Report) printErrors() {
...
+	for _, result := range misconfig.Results {
+		workloadMisconfigs := make([]types.DetectedMisconfiguration, 0)
+		infraMisconfigs := make([]types.DetectedMisconfiguration, 0)
+
+		for _, m := range result.Misconfigurations {
+			if strings.HasPrefix(m.ID, "KCV") {
+				infraMisconfigs = append(infraMisconfigs, m)
+			} else {
+				workloadMisconfigs = append(workloadMisconfigs, m)
+			}
```
Comment (chen-keinan) [inline pkg/k8s/report/report.go:303]: nit:
```suggestion
			if strings.HasPrefix(m.ID, "KCV") {
				infraMisconfigs = append(infraMisconfigs, m)
				continue
			} 
			workloadMisconfigs = append(workloadMisconfigs, m)
			
```

File: pkg/k8s/report/report.go [hunk 2]
```diff
@@ -168,34 +159,89 @@ func Write(report Report, option Option, securityChecks []string, showEmpty bool
...
+
+	return r
 }
 
 func rbacResource(misConfig Resource) bool {
 	return misConfig.Kind == "Role" || misConfig.Kind == "RoleBinding" || misConfig.Kind == "ClusterRole" || misConfig.Kind == "ClusterRoleBinding"
 }
 
+func infraResource(misConfig Resource) bool {
+	return misConfig.Kind == "Pod" && misConfig.Namespace == "kube-system"
```
Comment (chen-keinan) [inline pkg/k8s/report/report.go:242]: I'm using rancher desktop and did not got infra result at all , hope our checks take  under consideration these use cases

**Review by chen-keinan — CHANGES_REQUESTED** [review]

File: pkg/k8s/report/report.go (hunk 2 above)
Comment (josedonizetti) [reply pkg/k8s/report/report.go:235]: what do you see when you do `kubectl get pods -n kube-system` on rancher? (rancher is not working on my computer not sure why, but kind/minikube/kops/k3 are all based on kubeadm, which will have all k8s components under `kube-system`)

Comment (chen-keinan) [reply pkg/k8s/report/report.go:235]:
```
NAME                                      READY   STATUS    RESTARTS   AGE
coredns-7448499f4d-4bsnw                  1/1     Running   3          11d
metrics-server-86cbb8457f-xtfv7           1/1     Running   3          11d
local-path-provisioner-5ff76fc89d-wzw7x   1/1     Running   5          11d
```

Comment (josedonizetti) [reply pkg/k8s/report/report.go:235]: So rancher doesn’t expose the apiserver? Controller manager? Or it is in another ns?

**Comment by chen-keinan** [comment]
@josedonizetti could please relate issue to it.
also when running this (scan cluster for vuln only) command `trivy k8s --security-checks vuln --report summary cluster` 
I get no result.
```164 / 164 [----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 100.00% 781 p/s```
however when running the full cluster command I get complete result :
` k8s  --report summary cluster`
```
Summary Report for rancher-desktop
┌──────────────┬───────────────────────────────────┬──────────────────────────────┬────────────────────┬───────────────────┐
│  Namespace   │             Resource              │       Vulnerabilities        │ Misconfigurations  │      Secrets      │
│              │                                   ├─────┬──────┬──────┬─────┬────┼───┬───┬───┬────┬───┼───┬───┬───┬───┬───┤
│              │                                   │  C  │  H   │  M   │  L  │ U  │ C │ H │ M │ L  │ U │ C │ H │ M │ L │ U │
├──────────────┼───────────────────────────────────┼─────┼──────┼──────┼─────┼────┼───┼───┼───┼────┼───┼───┼───┼───┼───┼───┤
│ trivy-system │ Job/scan-vulnerabilityreport      │     │      │  1   │ 1   │ 2  │   │   │ 3 │ 4  │   │   │   │   │   │   │
│ profefe      │ Deployment/profefe                │  4  │  29  │  10  │ 2   │    │   │   │ 2 │ 10 │   │   │   │   │   │   │
│ kube-system  │ Deployment/metrics-server         │     │      │      │     │ 6  │   │   │ 3 │ 10 │   │   │   │   │   │   │
│ kube-system  │ Service/metrics-server            │     │      │      │     │    │   │   │ 1 │    │   │   │   │   │   │   │
│ kube-system  │ Deployment/coredns                │     │  7   │  6   │ 1   │ 5  │   │   │ 3 │ 5  │   │   │   │   │   │   │
│ kube-system  │ Deployment/local-path-provisioner │  4  │  32  │  10  │ 2   │    │   │   │ 3 │ 10 │   │   │   │   │   │   │
│ kube-system  │ Service/kube-dns                  │     │      │      │     │    │   │   │ 1 │    │   │   │   │   │   │   │
│ default      │ ReplicaSet/frontend               │ 565 │ 1372 │ 1157 │ 894 │ 64 │   │   │ 2 │ 10 │   │   │   │   │   │   │
│ default      │ Job/pi                            │ 61  │ 434  │ 368  │ 566 │ 4  │   │   │ 2 │ 10 │   │   │   │   │   │   │
└──────────────┴───────────────────────────────────┴─────┴──────┴──────┴─────┴────┴───┴───┴───┴────┴───┴───┴───┴───┴───┴───┘
```

**Comment by chen-keinan** [comment]
LGTM 🚀 I would check if needed to update some documentation , [example](https://aquasecurity.github.io/trivy/v0.31.3/docs/misconfiguration/policy/builtin/)

**Review by chen-keinan — APPROVED** [review]

**Comment by josedonizetti** [comment]
@chen-keinan @knqyf263 can you review? Did the change to support `--components`. If you agree with the design/code I'll add documentation next (maybe in a follow up PR), just to be sure this is what we want before going deeper.

**Comment by josedonizetti** [comment]
@chen-keinan @knqyf263 I really would like this to land on the next release. Let me know if anything is missing for the review.

**Comment by chen-keinan** [comment]
@josedonizetti its looks good 🚀  couple of question is the does the filter `--components=workload` or `--components=infra` works only if I explicitly define `--security-checks=config` flag ? 
for example if I put :
`k8s all --report=summary --namespace=kube-system  --components=infra` or
`k8s all --report=summary --namespace=kube-system  --components=workload`
I still get both ,  misconfiguration and infra checks on both commands:
```
 trivy k8s all --report=summary --namespace=kube-system  --components=workload
34 / 34 [--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 100.00% 2 p/s

Summary Report for minikube
┌─────────────┬──────────────────────────────────────┬─────────────────────┬────────────────────┬───────────────────┐
│  Namespace  │               Resource               │   Vulnerabilities   │ Misconfigurations  │      Secrets      │
│             │                                      ├───┬────┬───┬────┬───┼───┬───┬───┬────┬───┼───┬───┬───┬───┬───┤
│             │                                      │ C │ H  │ M │ L  │ U │ C │ H │ M │ L  │ U │ C │ H │ M │ L │ U │
├─────────────┼──────────────────────────────────────┼───┼────┼───┼────┼───┼───┼───┼───┼────┼───┼───┼───┼───┼───┼───┤
│ kube-system │ Pod/kube-controller-manager-minikube │   │    │   │    │   │   │ 1 │ 3 │ 8  │   │   │   │   │   │   │
│ kube-system │ Pod/kube-scheduler-minikube          │   │    │   │    │   │   │ 1 │ 3 │ 8  │   │   │   │   │   │   │
│ kube-system │ Pod/etcd-minikube                    │   │ 16 │ 4 │    │ 4 │   │ 1 │ 3 │ 7  │   │   │   │   │   │   │
│ kube-system │ Pod/kube-apiserver-minikube          │   │    │   │    │   │   │ 1 │ 3 │ 9  │   │   │   │   │   │   │
│ kube-system │ Pod/storage-provisioner              │   │ 8  │ 2 │    │ 3 │   │ 1 │ 5 │ 10 │   │   │   │   │   │   │
│ kube-system │ DaemonSet/kube-proxy                 │ 7 │ 11 │ 3 │ 56 │   │   │ 2 │ 4 │ 10 │   │   │   │   │   │   │
│ kube-system │ Service/kube-dns                     │   │    │   │    │   │   │   │ 1 │    │   │   │   │   │   │   │
│ kube-system │ Deployment/coredns                   │   │ 6  │ 2 │ 1  │ 5 │   │   │ 3 │ 5  │   │   │   │   │   │   │
└─────────────┴──────────────────────────────────────┴───┴────┴───┴────┴───┴───┴───┴───┴────┴───┴───┴───┴───┴───┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN


Summary Report for minikube
┌─────────────┬─────────────────────────────────────────────────────┬───────────────────┐
│  Namespace  │                      Resource                       │  RBAC Assessment  │
│             │                                                     ├───┬───┬───┬───┬───┤
│             │                                                     │ C │ H │ M │ L │ U │
├─────────────┼─────────────────────────────────────────────────────┼───┼───┼───┼───┼───┤
│ kube-system │ Role/system:persistent-volume-provisioner           │   │ 2 │   │   │   │
│ kube-system │ Role/system::leader-locking-kube-controller-manager │   │   │ 1 │   │   │
│ kube-system │ Role/system::leader-locking-kube-scheduler          │   │   │ 1 │   │   │
│ kube-system │ Role/system:controller:cloud-provider               │   │   │ 1 │   │   │
│ kube-system │ Role/system:controller:bootstrap-signer             │ 1 │   │   │   │   │
│ kube-system │ Role/system:controller:token-cleaner                │ 1 │   │   │   │   │
└─────────────┴─────────────────────────────────────────────────────┴───┴───┴───┴───┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN
```

**Comment by chen-keinan** [comment]
@josedonizetti looks good 🚀  , there is something strange experience wise when we run filter by workload we still get rbac results :
```
trivy k8s all --namespace=kube-system  --report=summary --components=workload
34 / 34 [--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 100.00% 2 p/s

Summary Report for minikube
┌─────────────┬──────────────────────────────────────┬─────────────────────┬────────────────────┬───────────────────┐
│  Namespace  │               Resource               │   Vulnerabilities   │ Misconfigurations  │      Secrets      │
│             │                                      ├───┬────┬───┬────┬───┼───┬───┬───┬────┬───┼───┬───┬───┬───┬───┤
│             │                                      │ C │ H  │ M │ L  │ U │ C │ H │ M │ L  │ U │ C │ H │ M │ L │ U │
├─────────────┼──────────────────────────────────────┼───┼────┼───┼────┼───┼───┼───┼───┼────┼───┼───┼───┼───┼───┼───┤
│ kube-system │ Pod/etcd-minikube                    │   │ 16 │ 4 │    │ 4 │   │ 1 │ 3 │ 7  │   │   │   │   │   │   │
│ kube-system │ Pod/kube-apiserver-minikube          │   │    │   │    │   │   │ 1 │ 3 │ 9  │   │   │   │   │   │   │
│ kube-system │ Pod/kube-scheduler-minikube          │   │    │   │    │   │   │ 1 │ 3 │ 8  │   │   │   │   │   │   │
│ kube-system │ Deployment/coredns                   │   │ 6  │ 2 │ 1  │ 5 │   │   │ 3 │ 5  │   │   │   │   │   │   │
│ kube-system │ DaemonSet/kube-proxy                 │ 7 │ 11 │ 3 │ 56 │   │   │ 2 │ 4 │ 10 │   │   │   │   │   │   │
│ kube-system │ Service/kube-dns                     │   │    │   │    │   │   │   │ 1 │    │   │   │   │   │   │   │
│ kube-system │ Pod/kube-controller-manager-minikube │   │    │   │    │   │   │ 1 │ 3 │ 8  │   │   │   │   │   │   │
│ kube-system │ Pod/storage-provisioner              │   │ 8  │ 2 │    │ 3 │   │ 1 │ 5 │ 10 │   │   │   │   │   │   │
└─────────────┴──────────────────────────────────────┴───┴────┴───┴────┴───┴───┴───┴───┴────┴───┴───┴───┴───┴───┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN


Summary Report for minikube
┌─────────────┬─────────────────────────────────────────────────────┬───────────────────┐
│  Namespace  │                      Resource                       │  RBAC Assessment  │
│             │                                                     ├───┬───┬───┬───┬───┤
│             │                                                     │ C │ H │ M │ L │ U │
├─────────────┼─────────────────────────────────────────────────────┼───┼───┼───┼───┼───┤
│ kube-system │ Role/system::leader-locking-kube-scheduler          │   │   │ 1 │   │   │
│ kube-system │ Role/system:controller:cloud-provider               │   │   │ 1 │   │   │
│ kube-system │ Role/system:controller:bootstrap-signer             │ 1 │   │   │   │   │
│ kube-system │ Role/system:persistent-volume-provisioner           │   │ 2 │   │   │   │
│ kube-system │ Role/system::leader-locking-kube-controller-manager │   │   │ 1 │   │   │
│ kube-system │ Role/system:controller:token-cleaner                │ 1 │   │   │   │   │
└─────────────┴─────────────────────────────────────────────────────┴───┴───┴───┴───┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN
```

**Comment by chen-keinan** [comment]
is the `--components=infra` is not supported on specific resource scan ? 
 ```trivy k8s Pod/kube-apiserver-minikube  --namespace=kube-system   --components=infra```

result : 

` ... 	FATAL	unknown flag: --components`

File: pkg/k8s/scanner/scanner.go [hunk 3]
```diff
@@ -139,3 +136,8 @@ func (s *Scanner) filter(ctx context.Context, r types.Report, artifact *artifact
 
 	return report.CreateResource(artifact, r, nil), nil
 }
+
+func shouldScanVulns(securityChecks []string) bool {
```
Comment (chen-keinan) [inline pkg/k8s/scanner/scanner.go:140]: nit:
```suggestion
func shouldScanVulnsOrSecrets(securityChecks []string) bool {
```

File: pkg/scanner/local/scan.go [hunk 4]
```diff
@@ -530,6 +530,7 @@ func mergePkgs(pkgs, pkgsFromCommands []ftypes.Package) []ftypes.Package {
 	return pkgs
 }
 
-func shouldScanMisconfig(securityChecks []string) bool {
-	return slices.Contains(securityChecks, types.SecurityCheckConfig) || slices.Contains(securityChecks, types.SecurityCheckRbac)
+func ShouldScanMisconfig(securityChecks []string) bool {
```
Comment (chen-keinan) [inline pkg/scanner/local/scan.go:533]: nit:
```suggestion
func ShouldScanMisconfigOrRbac(securityChecks []string) bool {
```

**Comment by chen-keinan** [comment]
Yes , I did run all tests on your branch , but I can check again

**Comment by josedonizetti** [comment]
> @josedonizetti looks good rocket , there is something strange experience wise when we run filter by workload we still get rbac results :

I thought this was the idea. I'm treating `workload` as everything that is not `infra`, as `rbac` isn't infra, it returns on the scanning. Should it be different?

**Comment by chen-keinan** [comment]
> > @josedonizetti looks good rocket , there is something strange experience wise when we run filter by workload we still get rbac results :
> 
> I thought this was the idea. I'm treating `workload` as everything that is not `infra`, as `rbac` isn't infra, it returns on the scanning. Should it be different? 

Yes ,in a way , but it could be confusing.
Anyway it should not hold the pr , we can discuss it later

**Comment by chen-keinan** [comment]
LGTM 🚀  well done !!

**Comment by josedonizetti** [comment]
Follow up issues:

- https://github.com/aquasecurity/trivy/issues/3055
- https://github.com/aquasecurity/trivy/issues/3056

**Review by knqyf263 — COMMENTED** [review]
Just to clarify my understanding, it filters out the misconfiguration results by AVD-ID when the resource is `Pod` under `kube-system` so it can know infra components, right?

**Comment by josedonizetti** [comment]
> Just to clarify my understanding, it filters out the misconfiguration results by AVD-ID when the resource is `Pod` under `kube-system` so it can know infra components, right?

Exactly!

**Comment by knqyf263** [comment]
I feel like we may want to show what results each table is displaying. It currently says `Summary Report for minikube` for all the tables, and I didn't understand how those tables are different at first.

**Comment by josedonizetti** [comment]
```
Summary Report for minikube

Workload Assessment
┌─────────────┬──────────────────────────────────────┬─────────────────────┬────────────────────┬───────────────────┐
│  Namespace  │               Resource               │   Vulnerabilities   │ Misconfigurations  │      Secrets      │
│             │                                      ├───┬────┬───┬────┬───┼───┬───┬───┬────┬───┼───┬───┬───┬───┬───┤
│             │                                      │ C │ H  │ M │ L  │ U │ C │ H │ M │ L  │ U │ C │ H │ M │ L │ U │
├─────────────┼──────────────────────────────────────┼───┼────┼───┼────┼───┼───┼───┼───┼────┼───┼───┼───┼───┼───┼───┤
│ kube-system │ Service/kube-dns                     │   │    │   │    │   │   │   │ 1 │    │   │   │   │   │   │   │
│ kube-system │ Pod/etcd-minikube                    │   │ 20 │ 4 │    │ 4 │   │ 1 │ 3 │ 7  │   │   │   │   │   │   │
│ kube-system │ Pod/kube-apiserver-minikube          │   │ 2  │ 1 │ 1  │ 2 │   │ 1 │ 3 │ 9  │   │   │   │   │   │   │
│ kube-system │ Pod/kube-scheduler-minikube          │   │ 2  │   │    │   │   │ 1 │ 3 │ 8  │   │   │   │   │   │   │
│ kube-system │ Pod/storage-provisioner              │   │ 9  │ 2 │    │ 3 │   │ 1 │ 5 │ 10 │   │   │   │   │   │   │
│ kube-system │ DaemonSet/kube-proxy                 │   │ 3  │ 2 │ 22 │   │   │ 2 │ 4 │ 10 │   │   │   │   │   │   │
│ kube-system │ Deployment/coredns                   │ 1 │ 3  │ 1 │ 1  │ 4 │   │   │ 3 │ 5  │   │   │   │   │   │   │
│ kube-system │ Pod/kube-controller-manager-minikube │   │ 2  │ 1 │ 1  │ 2 │   │ 1 │ 3 │ 8  │   │   │   │   │   │   │
└─────────────┴──────────────────────────────────────┴───┴────┴───┴────┴───┴───┴───┴───┴────┴───┴───┴───┴───┴───┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN


RBAC Assessment
┌─────────────┬─────────────────────────────────────────────────────┬───────────────────┐
│  Namespace  │                      Resource                       │  RBAC Assessment  │
│             │                                                     ├───┬───┬───┬───┬───┤
│             │                                                     │ C │ H │ M │ L │ U │
├─────────────┼─────────────────────────────────────────────────────┼───┼───┼───┼───┼───┤
│ kube-system │ Role/system:controller:bootstrap-signer             │ 1 │   │   │   │   │
│ kube-system │ Role/system:controller:cloud-provider               │   │   │ 1 │   │   │
│ kube-system │ Role/system:controller:token-cleaner                │ 1 │   │   │   │   │
│ kube-system │ Role/system::leader-locking-kube-scheduler          │   │   │ 1 │   │   │
│ kube-system │ Role/system::leader-locking-kube-controller-manager │   │   │ 1 │   │   │
│ kube-system │ Role/system:persistent-volume-provisioner           │   │ 2 │   │   │   │
└─────────────┴─────────────────────────────────────────────────────┴───┴───┴───┴───┴───┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN


Infra Assessment
┌─────────────┬──────────────────────────────────────┬─────────────────────────────┐
│  Namespace  │               Resource               │ Kubernetes Infra Assessment │
│             │                                      ├────┬────┬────┬─────┬────────┤
│             │                                      │ C  │ H  │ M  │ L   │   U    │
├─────────────┼──────────────────────────────────────┼────┼────┼────┼─────┼────────┤
│ kube-system │ Pod/kube-controller-manager-minikube │    │    │    │ 3   │        │
│ kube-system │ Pod/kube-scheduler-minikube          │    │    │    │ 1   │        │
│ kube-system │ Pod/kube-apiserver-minikube          │    │    │ 1  │ 10  │        │
└─────────────┴──────────────────────────────────────┴────┴────┴────┴─────┴────────┘
Severities: C=CRITICAL H=HIGH M=MEDIUM L=LOW U=UNKNOWN

```

**Comment by knqyf263** [comment]
@josedonizetti The test is failing now.

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**